### PR TITLE
Added transformer for pullrequests from bitbucket and github

### PR DESCRIPTION
**Pull request by borek2**
Because of the different json models that are responded from the Github and Bitbucket API, i made some transformers to be able to compare the models.

### CODE REVIEW

File: src/pullrequestTransformer.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		});
+	}
+	
+	function transformGithubPullrequest(pullrequest) {
+		var merged = (pullrequest.merged_at !== null);
+		return {
+			"title": pullrequest.title,
+			"author": pullrequest.user.login,
+			"created_on": pullrequest.created_at,
+			"updated_on": pullrequest.updated_at,
```
Comment (DaanvanderValk) [inline src/pullrequestTransformer.js]: Maybe use created_at instead of created_on? ;-)

Comment (borek2) [inline src/pullrequestTransformer.js]: I agree, will change, bitbucket uses "on" so that confused me ;)

**Comment by DaanvanderValk** [comment]
What is  src/.apicallers.js.swp?

**Comment by borek2** [comment]
Removed, shouldnt be there

**Pull request merged**